### ISSUE TITLE
About the reinforcement learning

### ISSUE THREAD

**Issue by LittleFlyingSheep** [XinhaoMei/DCASE2021_task6_v2#1]
Hi, thank you very much for your work!
I am reading your code with report to understand the work you did. How do you implement the reinforcement learning in "config.mode == 'finetune' "? Specifically, which part of the code is the 'CIDEr loss' part in 'Fine-tuning using RL'?

**Comment by XinhaoMei** [comment]
> Hi, thank you very much for your work!
> I am reading your code with report to understand the work you did. How do you implement the reinforcement learning in "config.mode == 'finetune' "? Specifically, which part of the code is the 'CIDEr loss' part in 'Fine-tuning using RL'?

Hi, the code for reinforcement learning is in finetune_rl.py. And you should set the corresponding settings in `rl` block in the setting file. Hope this helpful. Thanks!

**Comment by LittleFlyingSheep** [comment]
Thank you very much! It is my inattention that I only notice the 'train.py' but ignore the 'finetune.py'.
